library/std/src/sync/poison.rs

```diff
@@ -59,18 +51,19 @@
 //!   Note, however, that an `RwLock` may only be poisoned if a panic occurs
 //!   while it is locked exclusively (write mode). If a panic occurs in any reader,
 //!   then the lock will not be poisoned.
+//!
+//! Note that the [`Once`] type also employs poisoning, but since it has non-poisoning `force`
+//! methods available on it, we do not need to both create a separate `nonpoison::Once` type and
+//! also place [`Once`] in this module.
```

Comment:
Since this is user-facing the wording like "we need to create" is a bit odd, think this can be simplified

```suggestion
//! Note that the [`Once`] type also employs poisoning, but since it has non-poisoning `force`
//! methods available on it, there is no separate `nonpoison` and `poison` version.
```